test/Header/ListParser.php

```diff
@@ -0,0 +1,26 @@
+<?php
+/**
+ * @see       https://github.com/zendframework/zend-mail for the canonical source repository
+ * @copyright Copyright (c) 2005-2019 Zend Technologies USA Inc. (https://www.zend.com)
+ * @license   https://github.com/zendframework/zend-mail/blob/master/LICENSE.md New BSD License
+ */
+
+namespace ZendTest\Mail\Header;
+
+use PHPUnit\Framework\TestCase;
+use Zend\Mail\Header\ListParser;
+
+/**
+ * @covers Zend\Mail\Header\ListParser<extended>
+ */
+class ListParserTest extends TestCase
+{
+    public function testParseIgnoreQuoteDelimiterIfAlreadyInQuote()
+    {
+        $parsed = ListParser::parse('"john\'doe" <john@doe.com>,jane <jane@doe.com>');
+        $this->assertEquals($parsed, [
```

Comment:
Just a small one: we should swap order of the parameters in assertEquals, as the first is "expected" and the second "actual".

Reply:
You're right. I removed the test file entirely because there's another test which validates the correct parsing of the addresses.